### PR TITLE
Add export to sqlite feature

### PR DESCRIPTION
**Pull request by zhouzhuojie**
## Description

- Refactor DB related code from `repo` to `handler` package
- Add `export/sqlite` endpoint for exporting the whole database

## Motivation and Context

I want to setup Flagr for environment other than production. For example, the integration testing env, staging env, or just local unit test env. There's no easy way to copy flags from one flagr deploy to another, and we probably don't want to share the Flagr instances, their databases, or reuse the same Flagr polluting data recording. 

This PR builds an interface for exporting data to a simple sqlite file, which can be used for other Flagr instances.

I'm also thinking of serialization into some configuration files, like YAML or TOML. For example, `export/yaml`, and the files can be checked into code for testing or other environment. It may require some work for (de)serialization.

## How Has This Been Tested?
Locally. Unit tests coming soon.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: pkg/config/env.go [hunk 1]
```diff
@@ -9,10 +9,8 @@ var Config = struct {
 	// Port - Flagr server port
 	Port int `env:"PORT" envDefault:"18000"`
 
-	// GracefulCleanupTimeout - the timeout after graceful shutdown. It's useful to mitigate Kubernetes
-	// rolling update race condition.
-	GracefulCleanupTimeout time.Duration `env:"FLAGR_GRACEFUL_CLEANUP_TIMEOUT" envDefault:"2s"`
```
Comment (zhouzhuojie) [inline pkg/config/env.go]: not needed as the latest go-swagger supports graceful shutdown

File: pkg/config/config.go [hunk 2]
```diff
@@ -28,8 +28,12 @@ func init() {
 }
 
 func setupLogrus() {
+	l, err := logrus.ParseLevel(Config.LogrusLevel)
+	if err != nil {
+		logrus.WithField("err", err).Fatalf("failed to set logrus level:%s", Config.LogrusLevel)
```
Comment (zhouzhuojie) [inline pkg/config/config.go]: respect the env setting for log level

File: pkg/mapper/entity_restapi/e2r/e2r.go [hunk 3]
```diff
@@ -4,13 +4,12 @@ import (
...
 
 	"github.com/checkr/flagr/pkg/entity"
-	"github.com/checkr/flagr/pkg/repo"
 	"github.com/checkr/flagr/pkg/util"
 
 	"github.com/checkr/flagr/swagger_gen/models"
 )
 
-var getDB = repo.GetDB
+var getDB = entity.GetDB
```
Comment (zhouzhuojie) [inline pkg/mapper/entity_restapi/e2r/e2r.go]: refactor

File: swagger/export_sqlite.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+get:
+  tags:
+    - export
+  produces:
+    - application/octet-stream
+  responses:
+    200:
+      description: >
+        Export sqlite3 format of the db dump, which is converted from the
+        from the main database.
```
Comment (rstratton) [inline swagger/export_sqlite.yaml]: Duplicate "from the"

File: docs/api_docs/bundle.yaml [hunk 5]
```diff
@@ -734,6 +734,23 @@ paths:
...
+    get:
+      tags:
+        - export
+      produces:
+        - application/octet-stream
+      responses:
+        '200':
+          description: >
+            Export sqlite3 format of the db dump, which is converted from the
+            from the main database.
```
Comment (rstratton) [inline docs/api_docs/bundle.yaml]: Duplicate "from the"

**Pull request merged**